src/frontend/src/lib/components/ui/ButtonAsLink.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
```

Comment:
I suggest to rename this component to `ExternalLinkIcon`.

Reply:
Yes! Make sense, changed